src/OpenTelemetry.Exporter.Instana/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Changelog
 
-## Unreleased
+## 1.1.0
```

Comment:
I think we should probably release this version as 1.0.2 as these changes are mainly bug fixes.

Reply:
That works for me.  I was on the fence as to whether the new exception logging was really a minor feature.